lib/commands/context.js

```diff
@@ -225,7 +225,7 @@ helpers.startChromeSession = async function startChromeSession () {
   this.chromedriver = await this.setupNewChromedriver(opts, this.adb.curDeviceId, this.adb);
   this.chromedriver.on(Chromedriver.EVENT_CHANGED, (msg) => {
     if (msg.state === Chromedriver.STATE_STOPPED) {
-      this.onChromedriverStop(CHROMIUM_WIN);
+      await this.onChromedriverStop(CHROMIUM_WIN);
```

Comment:
this one won't work will it? since it's not in an async function?

Reply:
why? it is defined as async. Perhaps, the name of it is not the best one:

```
// Handle an out-of-band Chromedriver stop event
helpers.onChromedriverStop = async function (context) {
```